### DISCUSSION TITLE
Compatibilità con Debian

### DISCUSSION THREAD

**Discussion by pebbly93** [Supergiovane/node-red-contrib-knx-ultimate#332] [General]
Salve Max, complimenti per la libreria. L'ho testata su Node-red eseguito in Windows10, ma avrei la necessità di implementarla su sistema Debian. Ho già verificato, e le versioni di Node-red e node.js sono compatibili. Puoi assicurarmi che funzioni correttamente su Debian11? L'hai mai testata?

**Comment by Supergiovane** [comment]
Ciao Filippo
Non vedo particolari problemi. Basta avere node.js > 16.0.0 e node-red > 3.1.1
Io faccio girare i miei node-red su VM debian (ma non mi ricordo la versione di debian) senza alcun problema.

**Reply by pebbly93** [reply]
Perfetto, grazie mille per la tempestività.
